boot/zephyr/Kconfig

```diff
@@ -28,6 +27,24 @@ partition=MCUBOOT_PAD
 partition-size=0x200
 source "${ZEPHYR_BASE}/../nrf/subsys/partition_manager/Kconfig.template.partition_size"
 
+config PM_PARTITION_SIZE_MCUBOOT
```

Comment:
These multiple definition of same property is not the most elegant.
I would see here the one, maybe using additional modified Kconfig template.
This might be reworked later,